### ISSUE TITLE
Cannot add peer

### ISSUE THREAD

**Issue by TheYellowArchitect** [ipfs/go-ipfs#9047]
### Checklist

- [X] My issue is specific & actionable.
- [X] I am not suggesting a protocol enhancement.
- [X] I have searched on the [issue tracker](https://github.com/ipfs/go-ipfs/issues?q=is%3Aissue) for my issue.

### Description

I want to send a file to a friend. P2P is perfect for this, no uploading to central server, I upload directly to my friend.

I open go-IPFS, and send him my peer ID, and everything else the GUI provides (local IP lol), it all fails.
With CLI, I can see it is easily possible (haven't tried yet, but I am certain its possible)

My point is, sending a file to a friend is a basic usage for a P2P network (distributed or decentralized, doesnt matter)
Having a public IP, and public key of sorts, should be shown. Whatever string is required for a peer to add you to his network (or vice versa)

Generally, assuming we were both normies, we would instantly give up and not bother further. There is not even a tooltip or suggestion, once you paste a peer ID onto "Add Peer" button, and fail. Sure, it gives the formating it expects (that is great, unironically), but it should also tell you where to get it **without a terminal** (we both used windows - terminal doesn't recognize `ipfs`)

**Comment by Jorropo** [comment]
This very likely just a closed NAT issue.
Pls ask questions on https://discuss.ipfs.io/ instead.

> we both used windows - terminal doesn't recognize ipfs

That just a PATH issue.

**Comment by TheYellowArchitect** [comment]
>That just a PATH issue.

Isn't this a bug? @Jorropo 

>This very likely just a closed NAT issue.
Pls ask questions on https://discuss.ipfs.io/ instead.

I am confident (but not certain) I do not have closed NAT.
What string should I paste to connect to a peer? I literally was not even allowed to paste most strings I found. No public IP is displayed either, only local (127.0 and 192.168)
Weirdly, right now, 2 new IPs are in my adress list, but I cannot test with my friend right now.
tl;dr: What do I copy-paste onto "Add Connection"?

**Comment by Jorropo** [comment]
> Isn't this a bug?

No it's you that havn't correctly configured your terminal.

It might be a doc issue or a feature request (such as ipfs desktop automatically configure PATH for you), but it's not the right place for this (should be on ipfs desktop).

> I am confident (but not certain) I do not have closed NAT.

Post `ipfs id` then. (I want the `Adresses` section on the webui).

**Comment by Jorropo** [comment]
> What do I copy-paste onto "Add Connection"?

That just a question about how to use the software, that why I asked you to post on https://discuss.ipfs.io/ where more comunity peoples are active and will answer you.

Anyway should be a full multiaddr (from ip to peerid).
And a public IP is needed if you aren't on the same network:
For example the one of my server:
```
/ip4/163.172.31.60/tcp/4001/p2p/12D3KooWAuMTLB3XF8MJKRLqNxZF5cTjHP29LrhCzsr5rAqvRXHG
```
